### ISSUE TITLE
Improve and clarify our `infos/supply/...` endpoints

### ISSUE THREAD

**Issue by tdroxler** [alephium/explorer-backend#460]
Currently all our supply info are following CMC computation, but for example the `locked-alph` doesn't include the locked alph of the reserved addresses.

We need maybe to rename some values or at least the description of the endpoints to be clear.

We could also have new endpoints to return the ALL locked alph, regardless of the reserved addresses.

**Comment by MaudSimon** [comment]
I'll throw in some ideas on potential renaming of the current `locked-alph` and a description. 

- Rename ideas: `cmc-locked-alph `, `circ-locked-alph` `non-reserved-locked-alph`

- Description: Get the locked ALPH supply excluding reserved addresses' locked ALPH balance. This is used for circulation supply calculation in line with CMC methodology to avoid double counting. 

I do think having an endpoint that actually shows ALL the locked alph would be good as a lock of people instinctively would calculate circulating supply as total-locked. I also do think generally speaking that's a good info to have available easily even for us. And that endpoint I'd call `locked-alph` or maybe `total-locked-alph` to avoid any breaking change for people who were using the endpoint previously called `locked-alph`

**Comment by lowzyyy** [comment]
Not just locked alph. If we are going to change/add endpoints it could be `cmc-locked`, `cmc-circulating` and `locked`, `circulating`.
Cmc prefix would just not include those addresses as before, and endpoints without a prefix would include this addresses.
I think that coinmarketcap should not dictate the values of endpoints and they should comply with Alephium, not the other way around

**Comment by tdroxler** [comment]
I agree it would be nice if CMC was following our values, on the very first version we proposed them our endpoints, but they have clear and precise bullet points on how those values should be computed, if you don't follow that, you can forget to get validated by them. 

Maybe if we were bitcoin we could make change their mind.

I'm thinking out loud, but whatever endpoints we add, at least on  https://explorer.alephium.org/ we will still show the same values as CMC, otherwise we'll get daily ton of questions on why it's different. 

What we could do is to have a link on the `supply` box and it would open a pop up, same as the txs, where we precise the different kind of locked alph.

**Comment by MaudSimon** [comment]
@tdroxler agreed on the explorer page showing the same value as CMC.
